src/main/groovy/myFirstApp/AppModule.groovy

```diff
@@ -0,0 +1,24 @@
+package myFirstApp
+
+
+import com.todo.modules.DataAccessModule
+import com.todo.dao.TaskDaoI
+import com.todo.dao.impl.TaskDaoImpl
+import com.todo.service.TaskServicesI
+
+import com.todo.service.impl.TaskServiceImpl
+import groovy.transform.CompileStatic
+import restling.guice.modules.RestlingApplicationModule
+
+@CompileStatic
+class AppModule extends RestlingApplicationModule{
+    Class<MyAppRouter> routerClass = MyAppRouter
+
+    @Override
+    void configureCustomBindings() {
+//        bind(ConnectionI).to(ConnectionServiceImpl)
```

Comment:
Any code that is not required anymore should be removed instead of keeping it commented. This beats code readability and also unnecessarily increases size of code.